new-lamassu-admin/src/pages/Customers/helper.js

```diff
@@ -154,7 +187,13 @@ const EntryType = () => {
           <Field
             component={RadioGroup}
             name="requirement"
-            options={requirementOptions}
+            options={
+              requirementOptions
+              // TODO: Enable once custom info requirement manual entry is finished
+              // !R.isEmpty(customInfoRequirementOptions)
+              //   ? updateRequirementOptions(requirementOptions)
+              //   : requirementOptions
+            }
```

Comment:
This is related with accepting/rejecting custom information requests from the customer right?

Reply:
It will enable the operator to populate an existing custom info requirement. If there's none available, simply won't display it as an option